examples/simplecell/simplecell_model.py

```diff
@@ -0,0 +1,149 @@
+"""Run simple cell optimisation"""
```

Comment:
Is this duplicating the code from the notebook? I'd prefer to avoid that, otherwise we have two place to maintain things and to match the outputs.